src/workflow/ArcanistDiffWorkflow.php

```diff
@@ -1739,6 +1757,35 @@ private function getCommitMessageFromUser() {
   }
 
 
+  private function modifiesPath($paths_to_check, $changes) {
```

Comment:
would we we need an exclude regex as well or not needed as per our use case?

Reply:
it might be convenient later; let's add it when we actually need it, should be easy